src/main/java/net/dv8tion/jda/internal/utils/Helpers.java

```diff
@@ -72,6 +72,10 @@ public static int countMatches(final CharSequence seq, final char c)
         return count;
     }
 
+    public static String sanitizeChannelName(String input) {
+        return input.replace(' ', '-');
```

Comment:
If the input should be sanitized (instead of leaving it up to Discord / throwing on whitespace), I would recommend `String#replaceAll("\\s+", "-")` instead since then also tabs will be replaced and multiple spaces will just result in one dash and not one for each space.